src/masternodes/operators.h

```diff
@@ -33,9 +33,9 @@ struct CCreateOperatorMessage {
     uint8_t operatorState{0};
 
     CCreateOperatorMessage(CScript operatorAddress_, std::string operatorName_, std::string operatorURL_, OperatorState operatorState_)
-                            :  operatorAddress(operatorAddress_)
-                            ,  operatorName(operatorName_)
-                            ,  operatorURL(operatorURL_)
+                            :  operatorAddress(std::forward<CScript>(operatorAddress_))
+                            ,  operatorName(std::forward<std::string>(operatorName_))
+                            ,  operatorURL(std::forward<std::string>(operatorURL_))
```

Comment:
It should be `move`. `forward` was introduced to handle a situation when you can have const and non-const ref say
```
temlpate <typename T>
void wrapper(T&& t)
```
In this context `t` can be type rvalue or lvalue, `forward` ensures no copies.
https://en.cppreference.com/w/cpp/utility/forward

Reply:
https://github.com/DeFiCh/ain/pull/366#discussion_r629113157